core/src/main/java/com/datastax/oss/driver/internal/core/tracker/NoopRequestTracker.java

```diff
@@ -72,7 +75,8 @@ public void onNodeSuccess(
       long latencyNanos,
       @NonNull DriverExecutionProfile executionProfile,
       @NonNull Node node,
-      @NonNull String requestPrefix) {
+      @NonNull String requestPrefix,
+      @NonNull ExecutionInfo executionInfo) {
```

Comment:
This also needs an update for onNodeError() with an Executioninfo param

Reply:
Done.